packages/contracts-hyperlane/src/registry.ts

```diff
@@ -0,0 +1,128 @@
+import { GithubRegistry, IRegistry } from "@hyperlane-xyz/registry";
+import { Address } from "viem";
+
+type MaybePromise<T> = T | Promise<T> | PromiseLike<T>;
+
+/** Hyperlane Registry Chain Metadata (sdk type inference seems to break) */
+export interface HyperlaneChainMetadata {
+    chainId: string | number;
```

Comment:
Pretty sure only number?

Reply:
The SDK says "string | number" so aligns with that just to be safe